README.md

```diff
@@ -1162,19 +1160,12 @@ And of course the `Primus` instance as well.
 
 Heartbeats are used in Primus to figure out if we still have an active, working
 and reliable connection with the server. These heartbeats are sent from the
-**client** to the server.
-
-The heartbeats will only be sent when there is an idle connection, so there is
-very little to no overhead at all. The main reason for this is that we already
-know that the connection is alive when we receive data from the server.
+**server** to the client.
```

Comment:
Added the diagram. Did not specify that this is a reversal of how heartbeats works in 6.x.